.github/workflows/pytest.yml

```diff
@@ -0,0 +1,79 @@
+name: pytest
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  pytest:
+    strategy:
+      matrix:
+        python-version: ["3.6"]
+        os: [ubuntu-18.04]
+        floatx: [float32, float64]
+        test-subset:
+          - |
+            --ignore=pymc3/tests/test_examples.py
+            --ignore=pymc3/tests/test_distributions_random.py
+            --ignore=pymc3/tests/test_variational_inference.py
+            --ignore=pymc3/tests/test_shared.py
+            --ignore=pymc3/tests/test_smc.py
+            --ignore=pymc3/tests/test_updates.py
+            --ignore=pymc3/tests/test_posteriors.py
+            --ignore=pymc3/tests/test_sampling.py
+            --ignore=pymc3/tests/test_parallel_sampling.py
+            --ignore=pymc3/tests/test_dist_math.py
+            --ignore=pymc3/tests/test_distribution_defaults.py
+            --ignore=pymc3/tests/test_distributions_timeseries.py
+            --ignore=pymc3/tests/test_random.py
+            --ignore=pymc3/tests/test_gp.py
+            --ignore=pymc3/tests/test_shape_handling
+            --ignore=pymc3/tests/test_quadpotential.py
```

Comment:
I'm no pytest guru, so this maybe a trivial question: why are we ignoring these tests?

Reply:
it's to split them up between different jobs, which are run in parallel

it should match what's currently in `.travis.yml`

Reply:
so, [this job](https://github.com/pymc-devs/pymc3/pull/4254/checks?check_run_id=1453764481) runs all tests expect for some (which are ignored), [this job](https://github.com/pymc-devs/pymc3/pull/4254/checks?check_run_id=1453764499) runs some of the ones which the first job ignores, etc..

Reply:
Ah I see, thanks for the explanation!